fe/fe-core/src/main/java/com/starrocks/planner/SortNode.java

```diff
@@ -124,6 +133,36 @@ public SortInfo getSortInfo() {
     protected void computeStats(Analyzer analyzer) {
     }
 
+    public void buildRuntimeFilters(IdGenerator<RuntimeFilterId> generator, DescriptorTable descTbl) {
+        SessionVariable sessionVariable = ConnectContext.get().getSessionVariable();
+        // only support the runtime filter in TopN
+        if (limit + offset > SORT_RUNTIME_FILTER_THEHOLD || limit < 0
```

Comment:
why choose this threshold? even if the top-n is huge, I think the filter could still achieve some benefits?